clients/remote/remote.go

```diff
@@ -0,0 +1,97 @@
+package remote
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/influxdata/influx-cli/v2/api"
+	"github.com/influxdata/influx-cli/v2/clients"
+)
+
+type Client struct {
+	clients.CLI
+	api.RemoteConnectionsApi
+}
+
+type CreateParams struct {
+	Name             string
+	Description      string
+	OrgID            string
```

Comment:
Even though the API requires an org ID, most CLI commands support passing the org name. The command's impl has to look up the org ID as its 1st step in this case.